### PR TITLE
Check can-i-merge on each PR 

### PR DESCRIPTION
**Pull request by pjlee11**
This is the outcome of a learning day.

Resolves https://github.com/bbc/simorgh-infrastructure/issues/143

_Adds a new Jenkinsfile and a bash script for the new `can-i-merge` Jenkins job_

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: can-i-merge/bash.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    echo "Failed!"
+    exit 1
+  fi
+else
+  echo "Error: This is not the latest build"
+  echo "Failed!"
+  exit 1
+fi
+
+sleep 5
```
Comment (bcmn) [inline can-i-merge/bash.sh]: What is this waiting for?

File: can-i-merge/bash.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  echo "Error: This is not the latest build"
+  echo "Failed!"
+  exit 1
+fi
+
+sleep 5 
+
+lastBuildOutput2=$(curl $LAST_BUILD_JOB_URL --cert $COSMOS_CERT --key $COSMOS_KEY)
+
+if [[ $lastBuildOutput2 =~ "nextBuild\":null" ]]; then
```
Comment (dr3) [inline can-i-merge/bash.sh]: Would be better if this was functionalised to be less duplication

**Review by radiocontrolled — DISMISSED** [review]

File: can-i-merge/bash.sh (hunk 2 above)
Comment (pjlee11) [reply can-i-merge/bash.sh]: Tried that and it broke ... I'll try again 🌞

**Comment by jamesbhobbs** [comment]
PR closed and reopened to trigger Travis PR build

**Comment by jamesbhobbs** [comment]
Second thoughts I'm just closing this one.

**Pull request closed without merging**